From 790a74fea5ccc6a88a0dad309d9c737b98cfb35a Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Sun, 29 Aug 2010 01:23:11 +0100
Subject: [PATCH] rt{2860,2870}sta: Enable channels 12-14

These drivers attempt to limit the available radio channels according
to national regulations, but they currently have no mechanism for
setting the country so they always use the default limits.  Change the
default limits to allow use of channels 12-14.
---
 drivers/staging/rt2860/common/rt_channel.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/rt2860/common/rt_channel.c b/drivers/staging/rt2860/common/rt_channel.c
index 5387989..2b8508b 100644
--- a/drivers/staging/rt2860/common/rt_channel.c
+++ b/drivers/staging/rt2860/common/rt_channel.c
@@ -1405,8 +1405,8 @@ struct rt_ch_region ChRegion[] = {
 	 "",
 	 CE,
 	 {
-	  {1, 11, 20, BOTH, FALSE}
-	  ,			/* 2.4 G, ch 1~11 */
+	  {1, 14, 20, BOTH, FALSE}
+	  ,			/* 2.4 G, ch 1~14 */
 	  {36, 4, 20, BOTH, FALSE}
 	  ,			/* 5G, ch 52~64 */
 	  {52, 4, 20, BOTH, FALSE}
-- 
1.7.1

